src/client/components/hello/style.js

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components'
+
+export const Container = styled.div``
```

Comment:
Can you add little bit of styles here 
Like center , adding fonts and making the `hello` little bit bigger !
Else everything looks good !